test/e2e/flask/multichain-api/create-session.spec.ts

```diff
@@ -1,15 +1,27 @@
 import { strict as assert } from 'assert';
-import { withFixtures } from '../../helpers';
+import { largeDelayMs, WINDOW_TITLES, withFixtures } from '../../helpers';
 import { Driver } from '../../webdriver/driver';
 import FixtureBuilder from '../../fixture-builder';
+import { DEFAULT_FIXTURE_ACCOUNT } from '../../constants';
 import {
-  createSessionScopes,
+  confirmAndSwitchFocusToDapp,
+  initCreateSessionScopes,
   DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS,
   getSessionScopes,
   openMultichainDappAndConnectWalletWithExternallyConnectable,
+  getExpectedSessionScope,
+  addRequestAccountsToCreateSession,
+  assertOnlyRequestedNetworksAreSelected,
+  addAndAuthorizeAccount,
+  uncheckNetworksExceptMainnet,
 } from './testHelpers';
 
 describe('Multichain API', function () {
+  /**
+   * check {@link FixtureBuilder.withPreferencesControllerAdditionalAccountIdentities} for second injected account address.
+   */
+  const SECOND_INJECTED_ACCOUNT = '0x09781764c08de8ca82e156bbf156a3ca217c7950';
```

Comment:
I'm surprised there isn't a constant for this already. Well, not going to make you be the one to fix that